profiles/seccomp/default_linux.go

```diff
@@ -739,9 +739,12 @@ func DefaultProfile() *Seccomp {
 		},
 	}
 
+	// The DefaultErrnoRet is 1: EPERM
+	errnoRet := uint(1)
```

Comment:
Would still be nice to use the constant, then there is no need for a comment.

Reply:
Using the constant now, thanks!